### ISSUE TITLE
Better filter for RHEL/Centos -> EL in Platform Search

### ISSUE THREAD

**Issue by gregdek** [ansible/galaxy-issues#81]
From @robynbergeron comments in #15:

"My only concern here is that for people like me who type fast and hit enter -- I can type centos or rhel and it will come back with "no results" before the dropdown/autocomplete thing gets a chance to come up.  I almost wonder if there would also be a way to return an error message of sorts saying "searching for centos or RHEL? Try searching for EL instead" for the case where people type quick and hit enter to search."

**Comment by gregdek** [comment]
This has to do with fuzzy searching vs. exact searching, we will make the necessary changes in 1.1.1

**Comment by chouseknecht** [comment]
Fuzzy search applied to tags, platforms and authors. With fuzzy search the following things now work:
- type in a partial value and click return (as fast as you like, with no need to wait for autocomplete results) and relevant result will be returned.
- the platform search encompasses aliases and releases. For example, choose Platform and type in 'ora', hit return. All roles with a platform of 'EL' will be returned because 'oracle' is an alias for 'EL'.

This is ready for testing in QA.

**Comment by jlmitch5** [comment]
Verified you are not held up by fuzzy search.

**Comment by jlmitch5** [comment]
Typing in "rhel" "red hat" and "14.04" do not alias to the correct platform.

**Comment by jlmitch5** [comment]
verified "rhel", "red hat" and "oracle" now alias to the EL platform.  Version numbers for ubuntu-based systems do not alias, only the codenames for the releases, but that seems out of scope for this.  :+1:
